Client/AuthRequestsTesterViewController.swift

```diff
@@ -100,4 +103,39 @@ class AuthRequestsTesterViewController: UIViewController {
             }
         })
     }
+    
+    
+    private func adyenLogin() {
+        Karhoo.getUserService().login(userLogin: .init(username: "", password: "")).execute(callback: { [weak self] result in
+            if result.isSuccess() {
+                self?.responseLabel.textColor = .green
+                self?.responseLabel.text = "Success: \(result.isSuccess())"
+                self?.adyen()
+            } else {
+                self?.responseLabel.textColor = .red
+                self?.responseLabel.text = "Failed\nError: \(String(describing: result.errorValue().debugDescription))"
+            }
+        })
+    }
+    
+    private func adyen() {
```

Comment:
adyenPaymentdetails would be more accurate

Reply:
done